packages/core/src/render3/i18n.ts

```diff
@@ -969,6 +932,70 @@ function readUpdateOpCodes(
   }
 }
 
+function icuUpdateCase(
+    tView: TView, lView: LView, tIcuIndex: number, nodeIndex: number, bindingsStartIndex: number,
+    icus: TIcu[], caseCreated: boolean) {
```

Comment:
I was thinking if we could rename the `icus` argument to `tIcus` (here and in other extracted functions). In this case the `tIcuIndex` argument name would be more self-explanatory.
```suggestion
    tIcus: TIcu[], caseCreated: boolean) {
```

Reply:
done